src/axom/inlet/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ blt_list_append(TO inlet_sources ELEMENTS LuaReader.cpp IF SOL_FOUND)
 set(inlet_depends core slic sidre fmt)
 
 blt_list_append(TO inlet_depends ELEMENTS sol lua IF SOL_FOUND)
+blt_list_append( TO inlet_depends ELEMENTS cuda IF ENABLE_CUDA )
```

Comment:
This is very concerning along with a lot of the other places you added `cuda` as a dependency. Why was this added?

Reply:
I was getting a ton of errors about missing cuda raja methods/definitions/macros etc.  (100+ errors for an object file). 

I had seen the very same error but with raja hip definitions. The fix was to make targets that previously depended on **hip_runtime** to  now depend on **hip**.

I applied the same idea for the cuda errors, but I agree it's odd this is happening at all.

Reply:
FYI, these cuda changes were tested against the rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_cuda.cmake config, built with updated TPLs.

Reply:
@white238  
The cuda dependencies-everywhere problem is resolved  with a RAJA patch. (See LLNL/RAJA#1261 for patch contents)

This also fixes the problem of not defining "sol" when cuda is defined.